### PR TITLE
Update Linux install and comboFed examples - develop

### PR DESCRIPTION
**Pull request by trevorhardy**
### Summary
Small documentation changes to fix the instructions for installing under Ubuntu. Small changes to get the comboFederate example working to validate the installation.

Fixes issue #2263.

This is the "develop" version of PR #2266.

A similar PR has been made to hot fix these in main. This PR is being applied directly to main so that the public documentation will be fixed more quickly.

### CODE REVIEW

**Review by phlptp — APPROVED** [review]

**Comment by nightlark** [comment]
If this and #2266 are intended to be identical changes, we can close this PR and once the hotfix in #2266 is merged into main, sync the update to main into develop with a merge PR.

**Comment by trevorhardy** [comment]
@nightlark, if you're willing to do that I'm fine closing this one out and leaving it up to you to make sure its done correctly. Let me know.

File: docs/user-guide/installation/linux.md [hunk 1]
```diff
@@ -71,6 +78,28 @@ $ helics_recorder --version
...
+entered exec State
+message sent from fed1 to fed1/endpoint at time 1
+[2022-01-17 14:50:32.991] [console] [debug] fed1 (131072)[t=1e-09]::Granted Time=1e-09
+processed time 1e-09
+received message from fed1/endpoint at 0 ::message sent from fed1 to fed1/endpoint at time 1
+received updated value of 1 at 1e-09s from fed1/pub
+message sent from fed1 to fed1/endpoint at time 2
+...
+```
+
```
[the remark this answers is not in the document]
Comment (trevorhardy) [reply docs/user-guide/installation/linux.md:102]: Yes; good catch and thank you. I've added it to PR #2266.

**Comment by trevorhardy** [comment]
PR #2266 merged into main and will merge main into develop rather than this separate PR.

**Pull request closed without merging**